src/components/organisms/tabs/TalksTab/index.jsx

```diff
@@ -195,17 +202,36 @@ class Talks extends React.Component {
             })}
         </MDBRow>
         {this.state.showUpload && (
-          <UploadModal {...this.props} closeModal={this.handleUploadClose} />
+          <TalkUploadModal
+            {...this.props}
+            closeModal={this.handleUploadClose}
+          />
         )}
       </>
     );
   }
 }
 //#endregion
 
+//#region > Redux Mapping
+const mapStateToProps = (state) => ({
+  loggedUser: state.auth.loggedUser,
+  fetchedUser: state.user.fetchedUser,
+});
+
+const mapDispatchToProps = (dispatch) => {
+  return { deleteTalk: (talk) => dispatch(deleteTalkAction(talk)) };
+};
+//#endregion
+
 //#region > Exports
-//> Default Class
-export default withRouter(Talks);
+/**
+ * Provides its connected component with the pieces of the data it needs from
+ * the store, and the functions it can use to dispatch actions to the store.
+ * Got access to the history object’s properties and the closest
+ * <Route>'s match.
+ */
```

Comment:
```suggestion
/**
 * Provides its connected component with the pieces of the data it needs from
 * the store, and the functions it can use to dispatch actions to the store.
 *
 * Got access to the history object’s properties and the closest
 * <Route>'s match.
 */
```